packages/components/src/components/hds/copy/snippet/index.ts

```diff
@@ -15,9 +16,23 @@ export const SUCCESS_ICON = 'clipboard-checked';
 export const ERROR_ICON = 'clipboard-x';
 export const DEFAULT_STATUS = 'idle';
 
-export default class HdsCopySnippetIndexComponent extends Component {
+interface HdsCopySnippetSignature {
+  Args: {
```

Comment:
`onError`, `onSuccess` and `text` should come from the `hds-clipboard` modifier (they're forwarded to it)